lib/urn.rb

```diff
@@ -1,5 +1,6 @@
 class URN
-  PATTERN = 'urn:(?!urn:)[a-z0-9\-]{1,31}:[\S]+'.freeze
+  PATTERN = %(urn:(?!urn:)[A-Za-z0-9][A-Za-z0-9\-]{1,31}:) +
```

Comment:
Because that's the generic `PATTERN` (without any case-insensitive flag), should we be stricter here and change the `urn` to `[Uu][Rr][Nn]`?